cordova-plugin-inappbrowser.opam

```diff
@@ -2,9 +2,9 @@ opam-version: "2.0"
 version: "1.1"
 maintainer:   "Danny Willems <contact@danny-willems.be>"
 authors:      "Danny Willems <contact@danny-willems.be>"
-homepage:     "https://github.com/dannywillems/ocaml-cordova-plugin-inappbrowser"
-bug-reports:  "https://github.com/dannywillems/ocaml-cordova-plugin-inappbrowser/issues"
-dev-repo:     "https://github.com/dannywillems/ocaml-cordova-plugin-inappbrowser"
+homepage:     "git+https://dannywillems/ocaml-cordova-plugin-inappbrowser"
```

Comment:
Cette url n'est pas correcte. Il faut garder le `github.com`.

Reply:
Le "You should use URLs of the form "git://", "git+https://"" me semblait insinuer que le "github.com" était superflux autant pour moi je vais corriger tout ça.